libxmp/PSPBUILD

```diff
@@ -1,7 +1,7 @@
 pkgname=libxmp
 pkgver=4.5.0
 pkgrel=1
-pkgdesc="Libxmp is a library that renders module files to PCM data."
+pkgdesc="Libxmp is a library that renders module files to PCM data"
```

Comment:
I'm not a big fan on this particular change, because it isn't related to this PR. But since it's so small, we can leave it.